spec/requests/comments_spec.rb

```diff
@@ -3,6 +3,10 @@
 RSpec.describe 'comments', type: :request do
   path '/lessons/{lesson_id}/comments' do
     parameter name: 'lesson_id', in: :path, type: :string, description: 'lesson_id'
+    parameter name: 'page', in: :query, type: :integer, default: 1, required: false
```

Comment:
If these parameters you add relate only to **GET '/lessons/{lesson_id}/comments'** write them inside **get('list comments')** (after line 11) . Because now they affect both **GET '/lessons/{lesson_id}/comments'** and **POST '/lessons/{lesson_id}/comments'**. I guess You need query parameters only for GET in this path...

Reply:
Fixed